stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -269,6 +279,10 @@ public class MessageListViewModel(
         observeTypingUsers()
         observeMessages()
         observeChannel()
+
+        if (messageId != null) {
+            onOpenedFromPushNotification(messageId)
```

Comment:
Since the `messageId` can be passed even if not coming from a PN, maybe a different naming? something like `onMessageFocusRequested` or `onMessageRequested`? Also this and `scrollToMessage` have similar behavior could we refactor the logic here?

Reply:
Building on the [comment above](https://github.com/GetStream/stream-chat-android/pull/4628/files#r1090687325). We are using this only for push notifs due to certain limitations, so the preference here is to be highly indicative with regards to what this should be used for.

Also a minor note is that we actually don't scroll to the message and focus on it. We do that in `v6`, but here once again due to certain limitations we only open the respective channel/ thread. In any case this should be okay because we should redirect users to the latest messages in the channel.